client/lib/deploy.rs

```diff
@@ -202,46 +202,80 @@ mod tests {
     const ENTRYPOINT: &str = "entrypoint";
     const VERSION: &str = "0.1.0";
     const SAMPLE_DEPLOY: &str = r#"{
-        "hash": "5e5baa4f7c77233d1cabe8e28569e597eb5a00e62b29c3278d5179378551bb99",
-        "header": {
-          "account": "01f60bce2bb1059c41910eac1e7ee6c3ef4c8fcc63a901eb9603c1524cadfb0c18",
-          "timestamp": "2020-11-24T17:38:19.515Z",
-          "ttl": "10s",
-          "gas_price": 1,
-          "body_hash": "1edd9716bc3b94fb2e4bdc769a1bcb0b3c7c4df2135ff1c2a405f0ae22e47646",
-          "dependencies": [
-            "be5fdeea0240e999e376f8ecbce1bd4fd9336f58dae4a5842558a4da6ad35aa8",
-            "168d7ea9c88e76b3eef72759f2a7af24663cc871a469c7ba1387ca479e82fb41"
-          ],
-          "chain_name": "casper-test-chain-name-1"
-        },
-        "payment": {
-          "StoredVersionedContractByHash": {
-            "hash": "09dcee4b212cfd53642ab323fbef07dafafc6f945a80a00147f62910a915c4e6",
-            "version": null,
-            "entry_point": "entrypoint",
-            "args": "02000000070000006e616d655f3031010000000000070000006e616d655f3032040000002a00000001"
-          }
-        },
-        "session": {
-          "StoredVersionedContractByHash": {
-            "hash": "09dcee4b212cfd53642ab323fbef07dafafc6f945a80a00147f62910a915c4e6",
-            "version": null,
-            "entry_point": "entrypoint",
-            "args": "02000000070000006e616d655f3031010000000000070000006e616d655f3032040000002a00000001"
-          }
+      "hash": "4858bbd79ab7b825244c4e6959cbcd588a05608168ef36518bc6590937191d55",
+      "header": {
+        "account": "01f60bce2bb1059c41910eac1e7ee6c3ef4c8fcc63a901eb9603c1524cadfb0c18",
+        "timestamp": "2021-01-19T01:18:19.120Z",
+        "ttl": "10s",
+        "gas_price": 1,
+        "body_hash": "95f2f2358c4864f01f8b073ae6f5ae67baeaf7747fc0799d0078743c513bc1de",
+        "dependencies": [
+          "be5fdeea0240e999e376f8ecbce1bd4fd9336f58dae4a5842558a4da6ad35aa8",
+          "168d7ea9c88e76b3eef72759f2a7af24663cc871a469c7ba1387ca479e82fb41"
+        ],
+        "chain_name": "casper-test-chain-name-1"
+      },
+      "payment": {
+        "StoredVersionedContractByHash": {
+          "hash": "09dcee4b212cfd53642ab323fbef07dafafc6f945a80a00147f62910a915c4e6",
+          "version": null,
+          "entry_point": "entrypoint",
+          "args": [
+            [
+              "name_01",
+              {
+                "cl_type": "Bool",
+                "serialized_bytes": "00",
+                "parsed_to_json": false
```

Comment:
Seems oddly self-referential in name.  Maybe just `parsed`?

Reply:
in this context, I wouldn't blink if you just called it "value".

Reply:
Renamed.